### PR TITLE
fix(sdk): removed the content-length restriction on file upload

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. metriport/metriport#960

### Description

- Removed the size restriction on file upload
- Added alert if file is too large

### Testing

- To test the file upload, use the `/src/document/document-upload.ts`

### Release Plan

- Merge this
- Test on develop
- Release

### CODE REVIEW

File: packages/core/src/external/aws/lambda-logic/document-uploader.ts [hunk 1]
```diff
@@ -50,6 +51,12 @@ export async function documentUploaderHandler(
   // Get file info from the copied file
   const { size, contentType } = await s3Utils.getFileInfoFromS3(destinationKey, destinationBucket);
 
+  if (size && size > MAXIMUM_FILE_SIZE) {
+    // #1207 TODO: Delete the file if it's too large and alert the customer.
+    const message = "File size exceeds the maximum allowed size of 50 MB";
+    console.log(`${message}: ${size}`);
+    throw new MetriportError(message, null, { size });
```
Comment (leite08) [inline packages/core/src/external/aws/lambda-logic/document-uploader.ts:58]: This should just be a notification for now, right?

File: packages/lambdas/src/document-uploader.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const handler = async (event: S3Event) => {
     } catch (error) {
       console.log("Error in documentUploaderHandler", error);
       capture.error(error, {
-        extra: { context: `documentUploaderHandler`, error },
+        extra: { context: `documentUploaderHandler`, sourceBucket, sourceKey },
```
Comment (leite08) [inline packages/lambdas/src/document-uploader.ts:35]: shouldn't we keep sending the error on additional?

**Review by leite08 — APPROVED** [review]
Approving considering we can adjust the comments I made. Happy discuss if you don't think they should be addressed.

File: packages/core/src/external/aws/lambda-logic/document-uploader.ts [hunk 3]
```diff
@@ -68,6 +63,12 @@ export async function documentUploaderHandler(
   try {
     console.log("Forwarding call to server with this fileData:", JSON.stringify(fileData));
     await forwardCallToServer(cxId, apiServerURL, fileData);
+    if (size && size > MAXIMUM_FILE_SIZE) {
+      // #1207 TODO: Delete the file if it's too large and alert the customer.
+      const message = `File size exceeds the maximum allowed size of ${MAXIMUM_FILE_SIZE_MB} MB.`;
```
[the remark this answers is not in the document]
Comment (RamilGaripov) [reply packages/core/src/external/aws/lambda-logic/document-uploader.ts]: Right..

**Pull request merged**